packages/clarity-decode/src/interaction.ts

```diff
@@ -69,12 +69,14 @@ export function decode(tokens: Data.Token[]): InteractionEvent {
             };
             return { time, event, data: submitData };
         case Data.Event.Scroll:
+            const topHash = tokens.length > 5 ? (tokens[5] as string).split(Data.Constant.Dot) : null;
+            const bottomHash = tokens.length > 6 ? (tokens[6] as string).split(Data.Constant.Dot) : null;
             let scrollData: Interaction.ScrollData = {
                 target: tokens[2] as number,
                 x: tokens[3] as number,
                 y: tokens[4] as number,
-                top: tokens.length > 5 ? tokens[5] as string : null,
-                bottom: tokens.length > 6 ? tokens[6] as string : null
+                top: topHash?.length > 1 ? topHash[1] : null,
```

Comment:
>topHash?.length > 1 ? topHash[1] : null [](http://example.com/codeflow?start=21&length=39)

are you doing this to get the alpha hash instead of beta (or vice versa) ? You should upload both and handle that server side otherwise you'll break as soon as anyone starts flighting a new hashing algorithm